modules/ROOT/pages/index.adoc

```diff
@@ -41,13 +39,14 @@ When a service is called, Anypoint Service Mesh ensures a smooth request flow an
 image:service-mesh-request-flow.png[]
 
 . The client, which might be another microservice or any Kubernetes Ingress component, sends a request to the service.
-. Envoy captures the request and redirects it to the Istio-mixer.
-. The mixer adapter then calls the Anypoint Service Mesh adapter to perform policy checks and verifications.
-. The Anypoint Service Mesh adapter performs policy checks and responds back to the mixer.
-. The mixer communicates the outcome of the policy checks to the Envoy proxy.
-. When no policy violations are encountered, the request is routed to the microservice. The microservice runs the service logic and sends the response back to the client.
-. The adapter communicates with Anypoint Platform asynchronously to get the latest policies and contracts.
-. Finally, the adapter sends back the API analytics information to Anypoint Platform.
+. Envoy captures the request and redirects it to the Mule adapter using the Envoy filter.
+. The adapter then sends the request to Anypoint Platform.
+to perform policy checks and verifications.
```

Comment:
Please rollback this change. It was correct as it was.

Reply:
Done! Thanks!